src/index.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from 'react';
+import ReactDOM from 'react-dom';
+import "./css/bootstrap.min.css";
+import "./css/my-styles.css";
+
+const name = "World";
+
+
+// 1. -	React.createElement
```

Comment:
In real project you commonly should have 1 component in 1 file

Reply:
Splited, see the last commits